ocaml/idl/datamodel.ml

```diff
@@ -2187,7 +2187,13 @@ end
 module Tunnel = struct
   let tunnel_protocol =
```

Comment:
As we had discussed, `vxlan_mesh` added into `tunnel_protocol` field is a little confusing, or we can add some comments here to clarify.